js/src/main/scala/org/widok/Widget.scala

```diff
@@ -448,6 +451,32 @@ trait Widget[T] extends Node { self: T =>
     self
   }
 
+  def overflow(value: HTML.Overflow): T = { style.overflow := Some(value); self }
+  def overflow(value: ReadChannel[HTML.Overflow]): T = {
+    style.overflow << value.map(Some(_))
+    self
+  }
+
+  /** Trick from http://www.quirksmode.org/css/clearing.html instead of adding div().clearBoth() at the bottom */
+  def floatingContainer(): T = {
+    overflow(HTML.Overflow.Hidden)
+    width(Length.Percentage(1.0))
+    self
+  }
+
+  def float(value: HTML.Floating): T = { style.float:= Some(value); self }
+  def float(value: ReadChannel[HTML.Floating]): T = {
+    style.float << value.map(Some(_))
+    self
+  }
+
+  def clear(value: HTML.Clear): T = { style.clear := Some(value); self }
+  def clear(value: ReadChannel[HTML.Clear]): T = {
+    style.clear << value.map(Some(_))
+    self
+  }
+  def clearBoth(): T = { clear(HTML.Clear.Both); self }
```

Comment:
Can we take out `floatingContainer` and `clearBoth`? I'd prefer if the API only provides basic operations.